lecture-notes/l1-intro-new.md

```
@@ -0,0 +1,439 @@
+---
+sidebar_position: 1
+lecture_number: 1
+title: Designs and Programming Languages
+image: /img/lectures/web/l1.png
+---
+Designs and Programming Languages
+================
+ 
+   * [ Python Source (.zip)](/code/lectures/l1-intro/python.zip)
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside link text

(MD039, no-space-in-links)